lnclient/lnd/lnd.go

```diff
@@ -720,15 +742,34 @@ func (svc *LNDService) GetOnchainBalance(ctx context.Context) (*lnclient.Onchain
 }
 
 func (svc *LNDService) RedeemOnchainFunds(ctx context.Context, toAddress string) (txId string, err error) {
-	return "", nil
+	resp, err := svc.client.SendCoins(ctx, &lnrpc.SendCoinsRequest{
+		Addr:    toAddress,
+		SendAll: true,
```

Comment:
We have an issue in LDK right now that this also sends anchor reserves. Is this the case in LND?

Reply:
Yes, tested SendCoins with 2,3,4 and 5 channels in my LND node with `SendAll` set to true. It still left 10k, 20k, 30k, 40k and 50k sats in each case